pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,213 @@
+# Group Step
+ 
+A *group* step groups jobs and other steps into logical groups, and allows displaying them within groups on the build page in Buildkite UI.    
+
+For example, your build pipelines in Buildkite are able to support an unlimited number of jobs. However, a large number of jobs on the build page can look messy. If pipelines contain 4-5 steps that can be logically grouped together, you can put them in a group using the `group` step.  
+
+The group step also helps manage dependencies between a collection of steps in a DAG (for example, "step X" depends on everything in "group Y").
+ 
+<section class="Docs__note">
+  <h3>Note</h3>
+  <p>In some cases, using group steps might break the DAG visualisation of a build.</p>
+</section>   
+
+{:toc}
+ 
+A group step can be defined in your pipeline settings or your [pipeline.yml](/docs/pipelines/defining-steps) file.  
+
+This is what a typical group step use looks like:
+
+```yml
+- group: ":broom: Linting"
+  key: "linting"
+  depends_on: docker
+  steps:
+    - name: ":rubocop: RuboCop"
```

Comment:
Gotta escape these, otherwise we get this

![image](https://user-images.githubusercontent.com/95874/140303180-d66aa952-f7bb-4ecd-b78a-ab38556bafd5.png)

Reply:
I guess :broom: is a standard emoji, and the others are custom, and so inserted as imges.

Reply:
🧹-ed 😁